webpack.config.js

```diff
@@ -59,6 +60,11 @@ const config = {
       chunks: ['gallery'],
       template: "./src/index.html",
       filename: "gallery.html"
+    }),
+    new HtmlWebPackPlugin({
+      chunks: ['staging'],
+      template: "./src/index.html",
+      filename: "staging.html"
```

Comment:
why call it staging? I don't get it :)

Reply:
What I was thinking, you draw your Battle Goals before your start the scenario.
And a staging room/area is a place where you prepare for something.
-> staging.
What name do you prefer?

Reply:
Sorry for forgetting to answer this. Your logic makes sense, but my goal was to have a web page _only_ for battle goals that we can give e.g. satire gaming. 

We should also add an "about" Link that displays all existing pages, assigns the correct copyright etc. (so that we're not lost in list of "secret" links etc)